pkg/config/viperconfig/viper.go

```diff
@@ -84,12 +83,6 @@ func (c *safeConfig) sendNotification(key string, oldValue, newValue interface{}
 	c.notificationChannel <- notification
 }
 
-func getCallerLocation(nbStack int) string {
```

Comment:
Linter is complaining about this function being unused. Removing it for now since from the history, it seems like its never been used.